components/ArticlePreview/ArticlePreview.tsx

```diff
@@ -124,6 +127,15 @@ const ArticlePreview: NextPage<Props> = ({
                 <>
                   <span aria-hidden="true">&middot;</span>
                   <span>{readTime} min read</span>
+                  {likes && (
```

Comment:
Oh interesting because you need a way to know how many likes an article has without clicking into it. That makes sense

Reply:
Why do we need to know the amount of likes before it's loaded? 🤔 I don't think it's shown on the card?

Reply:
@NiallJoeMaher 
> Why do we need to know the amount of likes before it's loaded? 🤔 I don't think it's shown on the card?

I added it to the card in this PR. 
if you take a look at the PR description you will see how it looks. 
Heart icon with the like count. If there are no likes it wont show.

I figured you wouldnt like this :rofl so I added it as a data attribute and can edit to remove it from the card.
The like count has to be included somewhere in the html, so playwright can grab it. For the sort by top test.